src/experimental/__p0009_bits/accessor_basic.hpp

```diff
@@ -45,20 +45,22 @@
 
 #include "macros.hpp"
 
-#include <cstddef>
+#include <cstddef> // ptrdiff_t
```

Comment:
Did you process these with clang-format?  If so, that is fine.  I'm also okay with adding a pragma to tell clang-format to ignore the file.  Ref: https://philwrightdev.wordpress.com/2014/10/01/making-clang-format-ignore-sections/

Reply:
Preserving our discussion on mattermost here: clang-format'ing this version of kokkos/mdspan appears to cause compiler errors. So format checking is now disabled for `experimental` with a `.clang-format`